apps/admin-x-activitypub/src/utils/posts.ts

```diff
@@ -127,3 +127,65 @@ export function mapPostToActivity(post: Post): Activity {
         to: ''
     };
 }
+
+export function mapActivityToPost(activity: Activity): Post {
+    const object = activity.object;
+    const actor = activity.actor;
+    const attributedTo = object.attributedTo;
+
+    return {
+        id: object.id,
+        type: object.type === 'Article' ? PostType.Article : PostType.Note,
+        title: object.name || '',
+        excerpt: object.preview?.content || '',
+        summary: object.summary || null,
+        content: object.content || '',
+        url: object.url || '',
+        featureImageUrl: object.image || null,
+        publishedAt: new Date(object.published || '').toISOString(),
+        likeCount: 0,
+        likedByMe: object.liked || false,
+        replyCount: object.replyCount || 0,
+        readingTimeMinutes: 0,
+        attachments: object.attachment || [],
+        author: {
+            id: attributedTo.id,
+            handle: getAccountHandle(
+                new URL(attributedTo.id).host,
+                attributedTo.preferredUsername
+            ),
+            name: attributedTo.name || '',
+            url: attributedTo.id,
+            avatarUrl: attributedTo.icon?.url || '',
+            followedByMe: attributedTo.followedByMe || false
+        },
+        authoredByMe: object.authored || false,
```

Comment:
[Current logic in FeedItem](https://github.com/TryGhost/Ghost/blob/4386ab5d271a07ef082378a52633f7c0b61bf61a/apps/admin-x-activitypub/src/components/feed/FeedItem.tsx#L387) suggests that we should be looking at `object.attributedTo` when it's an Announce activity